Resources/skeleton/app/SymfonyRequirements.php

```diff
@@ -778,4 +780,21 @@ protected function getRealpathCacheSize()
                 return (int) $size;
         }
     }
+
+    /**
+     * Defines PHP required version from Symfony version.
+     *
+     * @return string The PHP required version
+     */
+    protected function getPhpRequiredVersion()
+    {
+        $composerConfig = json_decode(file_get_contents(__DIR__.'/../composer.json'), true);
+        if (array_key_exists('symfony/symfony', $composerConfig['require'])) {
```

Comment:
This logic is indeed very fragile. Especially the `int` cast below. And people can use this bundle without `symfony/symfony` even if this is probably not that widespread. Anyway, as other parts of this requirement file requires that the vendor are installed, I would use the composer.lock file instead if present. If not, do nothing and wait for the user to fix the vendor installation requirement first.